std/fs/constants.ts

```diff
@@ -0,0 +1,4 @@
+const { build } = Deno;
+
+export const isWindows = build.os === "win";
+export const EOL = isWindows ? "\r\n" : "\n";
```

Comment:
I apologize for being so nit-picky about this... 

1. These 4 lines could be one: `export const EOL = Deno.build.os === "win" ? "\r\n" : "\n";`
2. Do we need a standalone module for this one constant? What about moving this to `std/fs/eol.ts` ?

Reply:
Hmm, `isWindows` is quite a popular export. Are you suggesting to get rid of it?

Reply:
@ry Sorry could you clarify what should be done here? You didn't acknowledge `isWindows` also being an export.

Reply:
Sorry - I’m just hesitant if because this PR is doing a couple of different things... I guess it looks fine - but I would prefer a patch that moves files around to only do that...

Reply:
@nayeemrmn Maybe you can reduce this patch to just the path renames...

Reply:
Yes, done just now.